language_bindings/rust/src/Runtime_bindings.rs

```diff
@@ -0,0 +1,300 @@
+
```

Comment:
Has this code passed through [rustfmt](https://github.com/rust-lang/rustfmt) or another similar tool? Is there a `clang-tidy` equivalent for Rust that's worth using? [Clippy](https://github.com/rust-lang/rust-clippy), maybe?

Any such tools should run in a GitHub Actions workflows. There's already one for clang-format that can be extended to this (and generally to other languages down the line). I'm not a Rustacean myself, but I just want to make sure that we're using the proper tools for Rust code quality at the outset.